
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

<title>SpaceTx: experimental and computational methods compartison for spatially resolved transcriptomics</title>
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Assistant:wght@200;400;600;700&display=swap" rel="stylesheet">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="../_static/style.css" />
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script crossorigin="anonymous" integrity="sha256-Ae2Vz/4ePdIu6ZyI/5ZGsYnb+m0JlOmKPjt6XZ9JJkA=" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
    <link rel="author" title="About these documents" href="../about.html" />
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" /> 
  </head><body>
  <nav id="main-navbar" class="navbar navbar-expand-lg navbar-light">
    <div class="container pt-2 pb-4">
      <a class="navbar-brand" href="../index.html"><img id="main-logo" src="/_static/spacetx_logo_web_black.svg"></a>
      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
    </div>
    <div id="main-menubar" class="container-fluid">
      <div class="container position-relative">
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
          <ul class="navbar-nav me-auto mb-2 mb-lg-0 justify-content-between">
            <li class="nav-item dropdown">
              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
                About
              </a>
              <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
                <li><a class="dropdown-item" href="../about.html">SpaceTx</a></li>
                <li><a class="dropdown-item" href="../people.html">People</a></li>
                <li><a class="dropdown-item" href="../spacejam.html">SpaceJam Hackathon</a></li>
              </ul>
            </li>
            <li><a class="nav-link" href="../data_analysis.html">Data and Analysis</a></li>
            <li><a class="nav-link" href="../results.html">Results and Code</a></li>
            <li><a class="nav-link" href="../contribute.html">Contributing Guidelines</a></li>
          </ul>
          <form class="d-flex mb-4" id="main-search" action="../search.html" method="get">
            <input class="form-control form-control-sm me-2" type="search" placeholder="Search" aria-label="Search" name="q">
            <button class="btn btn-success btn-sm" type="submit">Search</button>
          </form>
        </div>
      </div>
    </div>
  </nav>

    <div id="wrapper">
      <div id="main" class="container">
        <div id="content">
          
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <section id="experimental-methods">
<h1>Experimental Methods<a class="headerlink" href="#experimental-methods" title="Permalink to this headline">¶</a></h1>
<br>
<div class="table-responsive"><table class="table text-break docutils align-default">
<colgroup>
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Experimental Method</p></th>
<th class="head"><p>Multiplexed smFISH</p></th>
<th class="head"><p>Allen MERFISH</p></th>
<th class="head"><p>hybridization-based in situ sequencing</p></th>
<th class="head"><p>ExSeq</p></th>
<th class="head"><p>Visium</p></th>
<th class="head"><p>BaristaSeq</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><strong>Readout mechanism</strong></p></td>
<td><p>non-barcoded</p></td>
<td><p>barcoded</p></td>
<td><p>sequencing-based</p></td>
<td><p>sequencing-based</p></td>
<td><p>spatial barcoding, sequencing en mass</p></td>
<td><p>sequencing-based</p></td>
</tr>
<tr class="row-odd"><td><p><strong>Enzymatic Amplification</strong></p></td>
<td><p>no</p></td>
<td><p>no</p></td>
<td><p>yes</p></td>
<td><p>yes</p></td>
<td><p>no readout imaging</p></td>
<td><p>yes</p></td>
</tr>
<tr class="row-even"><td><p><strong>Publications</strong></p></td>
<td><p><a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/30392798">https://www.ncbi.nlm.nih.gov/pubmed/30392798</a></p></td>
<td></td>
<td><p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/2020.02.03.931618v1">https://www.biorxiv.org/content/10.1101/2020.02.03.931618v1</a></p></td>
<td><p><a class="reference external" href="https://doi.org/10.1126/science.aax2656">https://doi.org/10.1126/science.aax2656</a></p></td>
<td><p><a class="reference external" href="https://science.sciencemag.org/content/353/6294/78">https://science.sciencemag.org/content/353/6294/78</a>
<a class="reference external" href="https://www.10xgenomics.com/products/spatial-gene-expression/">https://www.10xgenomics.com/products/spatial-gene-expression/</a></p></td>
<td><p><a class="reference external" href="https://doi.org/10.1093/nar/gkx1206">https://doi.org/10.1093/nar/gkx1206</a></p></td>
</tr>
<tr class="row-odd"><td><p><strong>Number of genes</strong></p></td>
<td><p>22</p></td>
<td><p>254</p></td>
<td><p>human: 120,
mouse: 119</p></td>
<td><p>mouse: 42</p></td>
<td><p>Full transcriptome</p></td>
<td><p>76</p></td>
</tr>
<tr class="row-even"><td><p><strong>Area Imaged</strong></p></td>
<td><p>~3mm<sup>2</sup></p></td>
<td><p>~1.9mm<sup>2</sup></p></td>
<td><p>human: 26.6mm<sup>2</sup>,
mouse: 53.6mm<sup>2</sup></p></td>
<td><p>mouse: ~1mm<sup>2</sup></p></td>
<td><p>6.5mm<sup>2</sup></p></td>
<td><p>0.75 x 1.2 mm</p></td>
</tr>
<tr class="row-odd"><td><p><strong>Tissue Thickness</strong></p></td>
<td><p>10 μm</p></td>
<td><p>10 μm</p></td>
<td><p>10 μm</p></td>
<td><p>20 μm</p></td>
<td><p>10 μm</p></td>
<td></td>
</tr>
<tr class="row-even"><td><p><strong>Detection Sensitivity (high: same order of magnitude as smFISH, medium: ~1 OoM lowerthan smFISH, low: more than 1 OoM counts per cell below smFISH)</strong></p></td>
<td><p>high</p></td>
<td><p>medium</p></td>
<td><p>low</p></td>
<td><p>medium</p></td>
<td><p>n/a</p></td>
<td><p>low</p></td>
</tr>
<tr class="row-odd"><td><p><strong>Average Total Counts Per Cell</strong></p></td>
<td><p>225</p></td>
<td><p>919</p></td>
<td><p>human: 38.6,
mouse: 30</p></td>
<td><p>mouse: 177 (for cells passing QC)</p></td>
<td><p>n/a</p></td>
<td><p>70</p></td>
</tr>
<tr class="row-even"><td><p><strong>Cellular Spatial Resolution?</strong></p></td>
<td><p>yes</p></td>
<td><p>yes</p></td>
<td><p>yes</p></td>
<td><p>subcellular</p></td>
<td><p>no</p></td>
<td><p>yes</p></td>
</tr>
<tr class="row-odd"><td><p><strong>Reagent Cost Per Experiment (USD)</strong></p></td>
<td></td>
<td><p>500</p></td>
<td></td>
<td><p>500</p></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td><p><strong>Time Per Experiment</strong></p></td>
<td><p>1 week</p></td>
<td><p>24 hours imaging, 4 days sample prep after sectioning</p></td>
<td><p>&lt; 1 week</p></td>
<td><p>4 days (imaging); 1 week of sample prep prior</p></td>
<td><p>~1 day</p></td>
<td></td>
</tr>
</tbody>
</table>
</div><p>SpaceTx Consortium data includes the following experimental methods:</p>
<ul class="simple">
<li><p><strong>multiround smFISH</strong>: <a class="reference external" href="https://www.nature.com/articles/s41467-021-23807-4">SSAM paper</a></p></li>
<li><p><strong>In-situ sequencing</strong>: <a class="reference external" href="https://academic.oup.com/nar/article/48/19/e112/5912821#282922333">hybISS paper</a></p></li>
<li><p><strong>ExSeq</strong>: <a class="reference external" href="https://www.science.org/doi/10.1126/science.aax2656">ExSeq paper</a></p></li>
<li><p><strong>MERFISH</strong>: This unpublished MERFISH data is from a prototype pipeline with max projected z-stacks prior to decoding.</p></li>
<li><p><strong>BaristaSeq</strong>: <a class="reference external" href="https://academic.oup.com/nar/article/46/4/e22/4668654">BaristaSeq paper</a></p></li>
</ul>
<p>In addition to these image based spatial transcriptomics methods, we also analyzed data from a sequencing-based method:</p>
<ul class="simple">
<li><p><strong>Visium</strong>: Visium is a (commercial) sequencing-based technique, where spatially barcoded oligonucleotides with a poly-T sequence are used to capture mRNA from fresh frozen tissue samples. In the Visium platform, capture locations (hereafter; spots), containing the barcoded oligonucleotides, are arranged according by “orange-crate packing” forming an equidistant hexagonal grid.  Spots have a center to center distance of 100μm and a diameter of 55μm. A square region with sides of 6.5mm, containing 4992 spots, constitute the capture area  onto which tissue samples are attached. Brightfield images are taken of the tissue samples after staining with hematoxylin and eosin (HE-images). Count data, obtained after sequencing, can be computationally aligned to the  HE-images and related to morphological features.</p></li>
</ul>
</section>


            <div class="clearer"></div>
          </div>
        </div>
      </div>
        </div>
      </div>
    </div>

  <div id="footer">
    <div class="container">
      <ul>
<li class="toctree-l1"><a class="reference internal" href="../about.html">About SpaceTx</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../about.html#introduction">Introduction</a></li>
<li class="toctree-l2"><a class="reference internal" href="../about.html#related-projects-and-ongoing-work">Related Projects and Ongoing Work</a></li>
<li class="toctree-l2"><a class="reference internal" href="../about.html#commercial-systems-for-spatial-transcriptomics">Commercial Systems for Spatial Transcriptomics</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../data_analysis.html">Data and Analysis</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#data">Data</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#cell-type-identification-in-single-cell-rnaseq-data">Cell Type Identification in single cell RNASeq data</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#marker-gene-selection">Marker Gene Selection</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#experimental-methods">Experimental Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#cell-segmentation">Cell Segmentation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#data-formatting-standards-and-computational-pipelines">Data Formatting Standards and Computational Pipelines</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#data-visualization">Data Visualization</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../results.html">Results and Code</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../results.html#cell-type-mapping-methods">Cell Type Mapping Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="../results.html#publications">Publications</a></li>
<li class="toctree-l2"><a class="reference internal" href="../results.html#jupyter-notebooks">Jupyter Notebooks</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../contribute.html">Contributing Guidelines</a></li>
</ul>

      <p class="copyright">
        © 2022 SpaceTx consortium<br>
      </p>
    </div>
  </div>

  </body>
</html>