
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

<title>SpaceTx: experimental and computational methods compartison for spatially resolved transcriptomics</title>
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Assistant:wght@200;400;600;700&display=swap" rel="stylesheet">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="../_static/style.css" />
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script crossorigin="anonymous" integrity="sha256-Ae2Vz/4ePdIu6ZyI/5ZGsYnb+m0JlOmKPjt6XZ9JJkA=" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" /> 
  </head><body>
  <nav id="main-navbar" class="navbar navbar-expand-lg navbar-light">
    <div class="container pt-2 pb-4">
      <a class="navbar-brand" href="../index.html"><img id="main-logo" src="/_static/spacetx_logo_web_black.svg"></a>
      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
    </div>
    <div id="main-menubar" class="container-fluid">
      <div class="container position-relative">
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
          <ul class="navbar-nav me-auto mb-2 mb-lg-0 justify-content-between">
            <li><a class="nav-link" href="../people.html">People</a></li>
            <li><a class="nav-link" href="../spacejam.html">SpaceJam Hackathon</a></li>
            <li><a class="nav-link" href="../data_analysis.html">Data and Analysis</a></li>
            <li><a class="nav-link" href="../results.html">Results and Code</a></li>
            <li><a class="nav-link" href="../contribute.html">Contributing Guidelines</a></li>
          </ul>
          <form class="d-flex mb-4" id="main-search" action="../search.html" method="get">
            <input class="form-control form-control-sm me-2" type="search" placeholder="Search" aria-label="Search" name="q">
            <button class="btn btn-success btn-sm" type="submit">Search</button>
          </form>
        </div>
      </div>
    </div>
  </nav>

    <div id="wrapper">
      <div id="main" class="container">
        <div id="content">
          
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <section id="cell-segmentation">
<h1>Cell Segmentation<a class="headerlink" href="#cell-segmentation" title="Permalink to this headline">¶</a></h1>
<p>Cell segmentation associates mRNA spot locations with individual cells, creating transcriptomic profiles that make up the cell-by-gene table.</p>
<p><strong>SpaceTx Cell Segmentation Pipeline</strong></p>
<ol class="arabic simple">
<li><p>Feature-based nucleus segmentation based on DAPI is applied to stitched 2D volumes and consists of two steps: (i) foreground segmentation and (ii) its consecutive splitting into separate instances (nuclei). For the first task, simple thresholding, either by employing one of the many available thresholding methods or by fine-tuning the cutoff intensity value, was sufficient to segment the foreground in all the available data sets. Splitting of the foreground into separate instances is performed per connected component, which allows efficient parallelizable implementation of our method. Based on the observation that most nuclei have rather regular elliptical shape, we developed an approach inspired by the work of Bilgin et al. [1] than employs elliptic features to extract two types of information: (i) curvature maps whose local minima correspond to locations of separation lines between touching nuclei and (ii) markers cognitively describing shapes of the nuclei and defined as the regions with positive Gaussian curvature and negative mean curvature. Calculation of the curvature maps and the markers is guided by a scale parameter, one for each, the value of which is chosen experimentally based on the average nucleus size. The markers are consecutively turned into super-pixels by performing skeletonization on the foreground with subtracted markers region. Calculated super-pixels are progressively merged based on the inter-pixel similarity score for obtaining the final segmentation. The similarity between two neighbouring super-pixels is defined based on the average value of the curvature map in the region between the corresponding markers and is recalculated after each merging iteration. This process stops after the maximum value of the inter-pixel similarity does not exceed the predefined threshold (whose value was kept fixed for all the test data sets). The described algorithm that worked equally well on the entire variety of SpaceTX data, including DAPI channels of the fluorescence microscopy data as well as 10x Visium histopathology images.</p></li>
<li><p>Augmented Cell Segmentation using Baysor. Using the nuclear segementation as a prior for cell location, Baysor assigned mRNA spots to cells probabilistically, including cell size and gene composition.</p></li>
</ol>
</section>


            <div class="clearer"></div>
          </div>
        </div>
      </div>
        </div>
      </div>
    </div>

  <div id="footer">
    <div class="container">
      <ul>
<li class="toctree-l1"><a class="reference internal" href="../data_analysis.html">Data and Analysis</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#data">Data</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#cell-type-identification-in-single-cell-rnaseq-data">Cell Type Identification in single cell RNASeq data</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#marker-gene-selection">Marker Gene Selection</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#experimental-methods">Experimental Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#cell-segmentation">Cell Segmentation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#data-formatting-standards-and-computational-pipelines">Data Formatting Standards and Computational Pipelines</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#cell-type-mapping-and-combined-results">Cell Type Mapping and Combined Results</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#data-visualization">Data Visualization</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../results.html">Results and Code</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../results.html#cell-type-mapping-methods">Cell Type Mapping Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="../results.html#publications">Publications</a></li>
<li class="toctree-l2"><a class="reference internal" href="../results.html#jupyter-notebooks">Jupyter Notebooks</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../discussion.html">Discussion</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../discussion/about-spacetx.html">About SpaceTx</a></li>
<li class="toctree-l2"><a class="reference internal" href="../discussion/discussion.html">Introduction</a></li>
<li class="toctree-l2"><a class="reference internal" href="../discussion/related-projects.html">Related Projects and Ongoing Work</a></li>
<li class="toctree-l2"><a class="reference internal" href="../discussion/commercial-systems.html">Commercial Systems for Spatial Transcriptomics</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../contribute.html">Contributing Guidelines</a></li>
</ul>

      <p class="copyright">
        © 2022 SpaceTx consortium<br>
      </p>
    </div>
  </div>

  </body>
</html>