
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

<title>SpaceTx: experimental and computational methods compartison for spatially resolved transcriptomics</title>
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Assistant:wght@200;400;600;700&display=swap" rel="stylesheet">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="../_static/style.css" />
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script crossorigin="anonymous" integrity="sha256-Ae2Vz/4ePdIu6ZyI/5ZGsYnb+m0JlOmKPjt6XZ9JJkA=" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
    <link rel="author" title="About these documents" href="../about.html" />
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" /> 
  </head><body>
  <nav id="main-navbar" class="navbar navbar-expand-lg navbar-light">
    <div class="container pt-2 pb-4">
      <a class="navbar-brand" href="../index.html"><img id="main-logo" src="/_static/spacetx_logo_web_black.svg"></a>
      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
    </div>
    <div id="main-menubar" class="container-fluid">
      <div class="container position-relative">
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
          <ul class="navbar-nav me-auto mb-2 mb-lg-0 justify-content-between">
            <li class="nav-item dropdown">
              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
                About
              </a>
              <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
                <li><a class="dropdown-item" href="../about.html">SpaceTx</a></li>
                <li><a class="dropdown-item" href="../people.html">People</a></li>
                <li><a class="dropdown-item" href="../spacejam.html">SpaceJam Hackathon</a></li>
              </ul>
            </li>
            <li><a class="nav-link" href="../data_analysis.html">Data and Analysis</a></li>
            <li><a class="nav-link" href="../results.html">Results and Code</a></li>
            <li><a class="nav-link" href="../contribute.html">Contributing Guidelines</a></li>
          </ul>
          <form class="d-flex mb-4" id="main-search" action="../search.html" method="get">
            <input class="form-control form-control-sm me-2" type="search" placeholder="Search" aria-label="Search" name="q">
            <button class="btn btn-success btn-sm" type="submit">Search</button>
          </form>
        </div>
      </div>
    </div>
  </nav>

    <div id="wrapper">
      <div id="main" class="container">
        <div id="content">
          
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <section id="data-visualization">
<h1>Data Visualization<a class="headerlink" href="#data-visualization" title="Permalink to this headline">¶</a></h1>
<p>Spatial Transcriptomics data presents unique challenges for visualization.</p>
<ul>
<li><dl class="simple">
<dt>Visualization of processed spatial transcriptomics data</dt><dd><ul class="simple">
<li><dl class="simple">
<dt>Processed Spatial Transcriptomics data typically includes:</dt><dd><ul>
<li><p>Spot Tables: Spatial locations for all mRNA spots detected in the experiment.</p></li>
<li><p>Cell by Gene Tables: Cell locations and the total mRNA counts associated with each cell.</p></li>
<li><p>Overview Image: spatial context for high-resolution data. For example, SpaceTx Consortium data includes high resolution data of mouse VISp and the overview image includes a DAPI image of the entire coronal section containing the VISp data.</p></li>
</ul>
</dd>
</dl>
</li>
</ul>
</dd>
</dl>
<ul>
<li><dl class="simple">
<dt>Cell by Gene Visualization in Cytosplore</dt><dd><ul class="simple">
<li><p>Boudewijn Lelieveldt and the Cytosplore Team have created a project-specific Cytosplore viewer for interactive exploration of SpaceTx cell by gene data</p></li>
<li><p>download link:</p></li>
<li><p>walk-through or screenshot</p></li>
</ul>
</dd>
</dl>
</li>
<li><dl>
<dt>Segmentation-free data analysis and visualization</dt><dd><ul class="simple">
<li><p>The heterogeneity of gene expression in tissue can be explored without cell segmentation.</p></li>
<li><p>The SSAM method (<a class="reference external" href="https://www.nature.com/articles/s41467-021-23807-4">Nature Communications volume 12, Article number: 3545 (2021)</a>) applied to SpaceTx data creates an easily visualized landscape of gene expression that can highlight rare cell types or tissue organization.</p></li>
</ul>
<blockquote>
<div><a class="reference internal image-reference" href="../_images/to_be_modified_SSAM.png"><img alt="../_images/to_be_modified_SSAM.png" src="../_images/to_be_modified_SSAM.png" style="width: 800px;" /></a>
</div></blockquote>
</dd>
</dl>
</li>
</ul>
</li>
<li><dl class="simple">
<dt>Image Data Visualization</dt><dd><ul class="simple">
<li><p>Unlike traditional biological imaging, spatial transcriptomics image data is often uninterpretible on its own. For example, each (round,channel) image from a MERFISH experiment typically contains diffraction-limited spots from all copies of 10s of different mRNA species. However, an image of DAPI from the same experiment could be very useful for visualizing tissue context for processed spatial transcriptomics data.</p></li>
<li><p>Raw image data for spatial transcriptomics is frequently at the terabyte scale and visualization requires comprehension of the spatial and (round,channel) organization of the images.</p></li>
<li><p>Modern bioimage visualization tools such as <a class="reference external" href="https://github.com/google/neuroglancer">Neuroglancer</a> and <a class="reference external" href="https://napari.org/">Napari</a> can be used to visualize spatial transcriptomics image data, but only to the extent that image data format information is interpretable by the visualization tool. Visualization tools will become more interoperable when and if the bioimaging community converges on formats and metadata standards for multiround fluorescence experiments. Large community standards projects like  <a class="reference external" href="https://forum.image.sc/tag/ome-ngff">OME-NGFF</a> as well as smaller, nascent projects like <a class="reference external" href="https://github.com/tlambert03/useq-schema">useq-schema</a> could form the basis of a metadata standard that could be applied to spatial transcriptomics microscopy data to simplify interactive visualization of large scale image data.</p></li>
</ul>
</dd>
</dl>
</li>
</ul>
</section>


            <div class="clearer"></div>
          </div>
        </div>
      </div>
        </div>
      </div>
    </div>

  <div id="footer">
    <div class="container">
      <ul>
<li class="toctree-l1"><a class="reference internal" href="../about.html">About SpaceTx</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../about.html#introduction">Introduction</a></li>
<li class="toctree-l2"><a class="reference internal" href="../about.html#related-projects-and-ongoing-work">Related Projects and Ongoing Work</a></li>
<li class="toctree-l2"><a class="reference internal" href="../about.html#commercial-systems-for-spatial-transcriptomics">Commercial Systems for Spatial Transcriptomics</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../data_analysis.html">Data and Analysis</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#data">Data</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#cell-type-identification-in-single-cell-rnaseq-data">Cell Type Identification in single cell RNASeq data</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#marker-gene-selection">Marker Gene Selection</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#experimental-methods">Experimental Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#cell-segmentation">Cell Segmentation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#data-formatting-standards-and-computational-pipelines">Data Formatting Standards and Computational Pipelines</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#cell-type-mapping-and-combined-results">Cell Type Mapping and Combined Results</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis.html#data-visualization">Data Visualization</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../results.html">Results and Code</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../results.html#cell-type-mapping-methods">Cell Type Mapping Methods</a></li>
<li class="toctree-l2"><a class="reference internal" href="../results.html#publications">Publications</a></li>
<li class="toctree-l2"><a class="reference internal" href="../results.html#jupyter-notebooks">Jupyter Notebooks</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../contribute.html">Contributing Guidelines</a></li>
</ul>

      <p class="copyright">
        © 2022 SpaceTx consortium<br>
      </p>
    </div>
  </div>

  </body>
</html>